src/lib/healthCheck/subServiceHealth.js

```diff
@@ -56,6 +56,31 @@ const getSubServiceHealthDatastore = async () => {
   }
 }
 
+/**
+ * @function getProxyCacheHealth
+ *
+ * @description
+ *   Gets the health of the proxy cache by checking the connection to the cache.
+ *
+ * @returns Promise<SubServiceHealth> The SubService health object for the proxy cache
+ */
+const getProxyCacheHealth = async ({ request }) => {
```

Comment:
Can we avoid passing Hapi-specific _request_ object to this function, and pass only proxyCache?